docs/user-guide/ze-working-with-data-sets.md

```diff
@@ -116,27 +178,29 @@
 
    - Volume Serial
 
-6. Select the attribute you want to edit, provide the value in the **input box**, and press `Enter`.
+6. Select the attribute you want to edit, provide the value in the **input box**, and press the `Enter` key.
 7. (Optional) Edit the parameters of your data set.
-8. Select the **+ Allocate Data Set** option to create the data set and list it in the **Side Bar**.
+8. Select the **+ Allocate Data Set** option.
+   
+   The data set is created and listed under the profile in the **Side Bar**.
 
    ![Set Parameters](../images/ze/ZE-set-params.gif)
 
 ## Creating data sets and data set members
```

Comment:
It's a bit strange to cover creating a data set again (without telling them to create a partitioned data set). I would skip steps 2-5 and in step 6 tell the user to right-click on a partitioned data set and select "Create New Member",